DNN Platform/Modules/ResourceManager/View.ascx.cs

```diff
@@ -217,6 +215,7 @@ protected void Page_Load(object sender, EventArgs e)
             }
             catch (Exception exc)
             {
+                // Module failed to load
```

Comment:
Do we really need this comment ? The line right under it is pretty self explanatory...

Reply:
This was left over from the original code, didn't realize you removed it. I'll fix it in the next code push